editor/plugins/tiles/tile_data_editors.cpp

```diff
@@ -628,6 +628,9 @@ void GenericTilePolygonEditor::_base_control_gui_input(Ref<InputEvent> p_event)
 					int closest_point;
 					_grab_polygon_point(mb->get_position(), xform, closest_polygon, closest_point);
 					if (closest_polygon >= 0) {
+						// Prevents Godot from crashing by dragging and deleting a point at the same time
```

Comment:
```suggestion
						// Prevents crash by dragging and deleting a point at the same time.
```
Comment style, and also more concise

Reply:
Thank you, updated.